Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -47,6 +47,28 @@ public void ClearValue(BindablePropertyKey propertyKey)
 			ClearValue(propertyKey.BindableProperty, false);
 		}
 
+		public bool IsSet(BindableProperty targetProperty)
+		{
+			if (targetProperty == null)
+				throw new ArgumentNullException(nameof(targetProperty));
+
+			return GetContext(targetProperty) != null;
+		}
+
+		public bool IsDefault(BindableProperty targetProperty)
+		{
+			if (targetProperty == null)
```

Comment:
Is the null check here needed since Calling `IsSet` is doing the same work? Right now, the argument is being checked for null twice.